socketsotetseg/src/main/java/net/runelite/client/plugins/socketsotetseg/SotetsegPlugin.java

```diff
@@ -194,6 +197,7 @@ protected void startUp()
 		flashFlag = true;
 
 		overlayManager.add(overlay);
+		overlayManager.add(mazeOverlay);
```

Comment:
You'll also want to remove the overlay on shutdown

Reply:
fixed in c477616e29e9c5c391c1172d73c9aadd8c952091